packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -317,8 +317,11 @@ const attributeRouter = (
       try {
         validateAuthorization(ctx, [ADMIN_ROLE, API_ROLE]);
```

Comment:
When you perform a manual test of the route on your local machine, it will probably fail with `403` here when called from the `m2m-gateway`, role is missing 👇  (remember also to update `authorizedRoles` in the API test)

```suggestion
        validateAuthorization(ctx, [ADMIN_ROLE, API_ROLE, M2M_ADMIN_ROLE]);
```